kedro-datasets/kedro_datasets/api/api_dataset.py

```diff
@@ -17,27 +19,18 @@ class APIDataSet(AbstractDataSet[None, requests.Response]):
     """``APIDataSet`` loads the data from HTTP(S) APIs.
     It uses the python requests library: https://requests.readthedocs.io/en/latest/
 
-    Example usage for the
-    `YAML API <https://kedro.readthedocs.io/en/stable/data/\
+    Example usage for the `YAML API <https://kedro.readthedocs.io/en/stable/data/\
     data_catalog.html#use-the-data-catalog-with-the-yaml-api>`_:
 
     .. code-block:: yaml
 
         usda:
-          type: api.APIDataSet
-          url: https://quickstats.nass.usda.gov
-          params:
-            key: SOME_TOKEN,
-            format: JSON,
-            commodity_desc: CORN,
-            statisticcat_des: YIELD,
-            agg_level_desc: STATE,
-            year: 2000
-
-    Example usage for the
-    `Python API <https://kedro.readthedocs.io/en/stable/data/\
-    data_catalog.html#use-the-data-catalog-with-the-code-api>`_:
-    ::
+          type: api.APIDataSet url: https://quickstats.nass.usda.gov params:
+            key: SOME_TOKEN, format: JSON, commodity_desc: CORN, statisticcat_des: YIELD,
+            agg_level_desc: STATE, year: 2000
+
```

Comment:
Seems like something went wrong with formatting here. Can you revert this change?